src/prisma/docker-compose.yml

```diff
@@ -16,4 +16,14 @@ services:
             port: 5432
             user: gddvrypsomjjxh
             password: 43d231b94fec1baa5b211d1c89309552434c91146a7aa66cafc86c71170698a8
-            ssl: true
\ No newline at end of file
+            ssl: true
+    postgres-db:
+    image: postgres:10.3
+    restart: always
+    environment:
+      POSTGRES_USER: prisma
+      POSTGRES_PASSWORD: prisma
+    volumes:
+      - postgres:/var/lib/postgresql/data
+volumes:
+  postgres: ~
```

Comment:
I think we decided this bit was unnecessary to include.

Reply:
ill delete it.